tests/validate-json.rb

```diff
@@ -34,6 +36,7 @@
       puts "  expected: 'tfa' to contain '#{v['schema']['contains']['const']}'" if v['type'].eql?('contains')
     end
     status = 1
+    exit(status)
```

Comment:
Better go on with the next file instead of exiting early:
```suggestion
    next
```